src/pages/WelcomePage/WelcomePage.module.css

```diff
@@ -15,3 +15,15 @@
 .paddingBottomZero {
   padding-bottom: 0;
 }
+
+ion-content.themedBackground::part(background) {
```

Comment:
Note to self @eecavanna: Learn about [CSS shadow parts](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_shadow_parts).